docker-compose.yml

```diff
@@ -22,3 +22,4 @@ services:
       - LOOKER_API_CLIENT_SECRET
       - GITHUB_ACCESS_TOKEN
       - UPDATE_DEV_BRANCHES
+      - UPDATE_SPOKE_BRANCHES
```

Comment:
Does this need to be set?

Reply:
Yes, otherwise the [`UPDATE_SPOKE_BRANCHES` env variable](https://github.com/mozilla/telemetry-airflow/pull/1362/files#diff-67fcd8dc7d9c494d949102b2e5d1dc33e864059908bb5605144779f11101078fR156) that we set in the Airflow DAG would not get passed into the Docker container